osu.Game/Screens/OnlinePlay/Matchmaking/Screens/Pick/PickScreen.cs

```diff
@@ -11,23 +11,38 @@
 
 namespace osu.Game.Screens.OnlinePlay.Matchmaking.Screens.Pick
 {
-    public partial class PickScreen : OsuScreen
+    public partial class PickScreen : MatchmakingSubScreen
     {
-        private BeatmapSelectionGrid selectionGrid = null!;
+        public override PlayerPanelList.PanelDisplayStyle PlayersDisplayStyle => PlayerPanelList.PanelDisplayStyle.Split;
+        public override Drawable PlayersDisplayArea { get; }
+
+        private readonly BeatmapSelectionGrid selectionGrid;
 
         [Resolved]
         private MultiplayerClient client { get; set; } = null!;
 
-        [BackgroundDependencyLoader]
-        private void load()
+        public PickScreen()
```

Comment:
When going through this PR, you might notice that I've moved `load()` to the `ctor()`. This is on purpose because `PlayersDisplayArea` needs to be available immediately.

I _could_ do this with `LoadComplete` bindings and all that, but this is probably the least nuclear option.